src/Resources/contao/templates/backend/be_pagination.html5

```diff
@@ -1,22 +1,24 @@
 
 <div class="pagination <?= $this->class ?>">
 
-  <p><?= $this->total ?></p>
-
-  <ul>
-    <?php if ($this->hasFirst): ?>
-      <li class="first"><a href="<?= $this->first['href'] ?>" class="first" title="<?= $this->first['title'] ?>"><?= $this->first['link'] ?></a></li>
-    <?php endif; ?>
-    <?php if ($this->hasPrevious): ?>
-      <li class="previous"><a href="<?= $this->previous['href'] ?>" class="previous" title="<?= $this->previous['title'] ?>"><?= $this->previous['link'] ?></a></li>
-    <?php endif; ?>
-    <?= $this->items ?>
-    <?php if ($this->hasNext): ?>
-      <li class="next"><a href="<?= $this->next['href'] ?>" class="next" title="<?= $this->next['title'] ?>"><?= $this->next['link'] ?></a></li>
-    <?php endif; ?>
-    <?php if ($this->hasLast): ?>
-      <li class="last"><a href="<?= $this->last['href'] ?>" class="last" title="<?= $this->last['title'] ?>"><?= $this->last['link'] ?></a></li>
-    <?php endif; ?>
-  </ul>
+  <nav role="navigation" aria-label="<?= $GLOBALS['TL_LANG']['MSC']['paginationAriaLabel'] ?>">
```

Comment:
I would prefer to replace the surrounding `<div>` with the `<nav>` element instead of adding another nesting level.